modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -65,16 +65,11 @@
 	}
 }
 
-.lfr-ballon-editor-insert-button {
-	align-items: center;
+.lfr-balloon-editor-insert-button {
```

Comment:
I remember the plus of the plus button was not perfectly aligned and seeing this PR video I saw it fixed. This CSS is a part of it?

Reply:
Yep! It is still not perfect, but my goal in this PR was just for it not to look broken. It doesn't look identical to [mockups](https://www.figma.com/file/RiqHAWbLTuDYWjgNgcXWkg/lexi-1084---balloon-toolbar?node-id=116%3A27843), for example plus is too small. We might want to replace the `+` character with an icon, but, even if we want to, it doesn't look like a trivial task. I am not sure what's the best ways to use Clay icons in a ckeditor plugin.